src/Microsoft.AspNetCore.Mvc.Core/Authorization/AuthorizeFilter.cs

```diff
@@ -106,6 +107,11 @@ public AuthorizeFilter(string policy)
                 throw new ArgumentNullException(nameof(context));
             }
 
+            if (context.Result != null)
+            {
+                return;
```

Comment:
Ok is this just a bogus test then since its directly calling the filter which won't happen...

https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNetCore.Mvc.Core.Test/Authorization/AuthorizeFilterTest.cs#L308